docs/getting_started/00_how_to_use_windmill/index.mdx

```diff
@@ -56,20 +43,7 @@ From a [VS Code extension](../../cli_local_dev/1_vscode-extension/index.mdx) to
 		description="Build scripts and flows in the comfort of your VS Code Editor, while leveraging Windmill UIs for test & flows edition."
 		href="/docs/cli_local_dev/vscode-extension"
 	/>
-	<DocCard
-		title="Deploy to Prod"
-		description="Deploy to prod using a staging workspace"
-		href="/docs/advanced/deploy_to_prod"
-	/>
 </div>
 
-## Where to Start?
-
-Windmill is a feature-rich developer platform to build endpoints, cron jobs, workflows & UIs. Each of these features can be used standalone.
-
-However the 3 core layers of Windmill, namely Scripts, Flows and Apps are complementary. Scripts are the smallest "unit" of abstraction and workflows and UIs rely on them.
-
-- Have a look at our quickstarts (on [Scripts](../../getting_started/0_scripts_quickstart/1_typescript_quickstart/index.mdx), [Flows](../6_flows_quickstart/index.mdx) or [Apps](../7_apps_quickstart/index.mdx)) that give an introduction on how to build your first ones.
-- Check how to [trigger scripts](../8_trigger_scripts/index.mdx) and [trigger flows](../9_trigger_flows/index.mdx) to get an idea of what you could do with Windmill.
-- Play with Windmill: [Cloud App](#use-windmill-cloud) or [Self-Hosting](#self-host-windmill). Our cloud Demo workspace showcases some speaking examples.
-- Browse [Windmill Hub](https://hub.windmill.dev/) and explore templates shared by the community.
+### Ready to Deploy?
+Move from staging to production seamlessly with Windmill's deployment guides, ensuring your projects are production-ready. [Deploy to production](/docs/advanced/deploy_to_prod).
```

Comment:
by the way it's better to link with ../../ etc. rather than the URL because if the name of the file change the link will be broken and we will not know